### ISSUE TITLE
Will not start. column user_id is not unique and unable to open database

### ISSUE THREAD

**Issue by bwoodill** [Tautulli/Tautulli#1065]
**Version:** v1.4.21

**Branch:** Master

**Commit hash:**

**Operating system:** Server 2012r2

**Python version:** 2.7.11

**What you did?** Updated to latest version using update in web UI

**What happened?** Will not start. Has error about unable to open database after one about column user_id is not unique

**What you expected?** To start

**How can we reproduce your issue?**

**What are your (relevant) settings?**

**Link to logs:** https://pastebin.com/9KZRkkkr

**Comment by JonnyWong16** [comment]
Restore a backup database from the backups folder.

**Comment by bwoodill** [comment]
Thanks for the quick response. I already tried restoring the db file from the backup folder and couldn't get any of them to work. I also tried using sqlite to dump the database to a sql file and then back into a db file and no luck. I also tried integrity_check and foreign_key_check. 

Also, my Plex database seems to have corrupted around the same time (dashboard not loading). I couldn't repair the plex database after following their page on repairing corruption so went through my libraries disabling one by one until I found which one was causing it to fail. I am now going to re add the library and see if it doesn't crash the dashboard.
